src/app/components/main-page/main-page.component.ts

```diff
@@ -15,30 +18,61 @@ export class MainPageComponent implements OnInit {
   faSearch = faSearch;
   faUser = faUser;
   faHome = faHome;
+
   isSearchCollapsed = true;
   isAtHomePage = false;
   isAtProfilePage = false;
+  user: User;
 
   constructor(
     private firebaseAuth: AngularFireAuth,
-    private router: Router) {
+    private router: Router,
+    private eventsService: EventsService,
+    private toastr: ToastrService,
+    private firebaseStorage: AngularFireStorage) {
```

Comment:
Should all our constructors look this way? Just wondering...

Reply:
No i just normally do it this way because its simple and clean and the auto document formatter wont mess with it